### PR TITLE
[Easy] clean up the LCM docstrings.

### PR DESCRIPTION
**Pull request by sayakpaul**
Cc: @dg845 @nagolinc

### CODE REVIEW

File: docs/source/en/api/pipelines/latent_consistency_models.md [hunk 1]
```diff
@@ -8,47 +8,8 @@ The abstract of the [paper](https://arxiv.org/pdf/2310.04378.pdf) is as follows:
 
 A demo for the [SimianLuo/LCM_Dreamshaper_v7](https://huggingface.co/SimianLuo/LCM_Dreamshaper_v7) checkpoint can be found [here](https://huggingface.co/spaces/SimianLuo/Latent_Consistency_Model).
 
-This pipeline was contributed by [luosiallen](https://luosiallen.github.io/) and [dg845](https://github.com/dg845).
+The pipelines were contributed by [luosiallen](https://luosiallen.github.io/) and [dg845](https://github.com/dg845).
```
Comment (dg845) [inline docs/source/en/api/pipelines/latent_consistency_models.md:11]: Also credit @nagolinc here for their work on the img2img pipeline?

Comment (sayakpaul) [reply docs/source/en/api/pipelines/latent_consistency_models.md:11]: Right away. Thanks for flagging.

Comment (sayakpaul) [inline docs/source/en/api/pipelines/latent_consistency_models.md:11]:
```suggestion
The pipelines were contributed by [luosiallen](https://luosiallen.github.io/), [nagolinc](https://github.com/nagolinc), and [dg845](https://github.com/dg845).
```

**Comment by patrickvonplaten** [comment]
Thanks!

**Pull request merged**